tests/installation/select_visible_patterns.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Select given PATTERNS
+#    You can pass
+#    PATTERNS=minimal,base or
+#
+#    For this you need to have needles that provide base-pattern,
+#    minimal-pattern...
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+use utils 'type_string_slow';
+
+sub run {
+    my ($self) = @_;
+    $self->go_to_patterns();
+    $self->process_visible_patterns();
```

Comment:
you don't need existing function `process_visible_patterns` or as suggested `process_only_visible_patterns_from_top`, we can just have here both methods:
`select_not_install_any_pattern` and `select_current_visible_patterns` I think it is more clear, and only one of those actually have parameters.